src/views/QuestionPage.jsx

```diff
@@ -1,13 +1,23 @@
 import React, { Fragment } from 'react';
-import Preview from '../components/Preview';
-
+import { Helmet } from 'react-helmet';
+import StoryTemplate from '../views/StoryTemplate';
+import Header from '../components/Header';
+import Footer from '../components/Footer';
 
 const Markup = (props) => (
   <Fragment>
-    <h1>Questions Page</h1>
-    <Preview>
-      {JSON.stringify(props, 0, 2)}
-    </Preview>
+
+    <Helmet>
+      <link href="https://fonts.googleapis.com/css?family=Roboto" rel="stylesheet"/>
+      <link rel="stylesheet" href="https://fonts.googleapis.com/icon?family=Material+Icons" />
```

Comment:
Is there a specific reason why you are using the Material Icons Font and not the SVG one through `import`?

Reply:
No, would you prefer that I rather use the SVG one?